metadata/group-contacts.csv

```diff
@@ -98,8 +98,8 @@ Rfam,Ioanna Kalvari,@kalvari,
 Rfam,Joanna Arganiska,@JoannaArgasinska,
 RGD,Stan Laulederkind,@slaulederkind,
 RGD,Tom Hayman,@gthayman,
-RGD,Jennifer Smith,jrsjrs,pipeline and data
-RGD,Marek Tuta,tutajm,import and export pipelines
+RGD,Jennifer Smith,@jrsjrs,pipeline and data
+RGD,Marek Tuta,tutajm,@import and export pipelines
```

Comment:
@pgaudet I think you missed here...I don't think that's their username.

Reply:
thanks !